pkg/conthandler/container_watcher_interface.go

```diff
@@ -0,0 +1,5 @@
+package conthandler
+
+type containerWatcherClient interface {
+	StartWatchedOnNewContainers()
```

Comment:
```suggestion
	WatchNewContainers()
```
[TODO]: There is no need of stating `start` and `watch` in the same function name

Reply:
+1

Reply:
also error handling maybe?